packages/client/src/views/GrantActivity.vue

```diff
@@ -0,0 +1,167 @@
+          class="mx-1"
+        >&bull;</span>
+        <span v-if="showNotesSummary">{{ notesSummaryText }}</span>
+      </div>
+    </div>
+
+    <template #footer>
+      <!-- Feed -->
+    </template>
+  </b-card>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+export default {
+  components: {},
+  data() {
+    return {
+      userIsFollowing: false,
+      loadingFollowState: false,
+      followers: [],
+      notes: [],
+    };
+  },
+  computed: {
+    ...mapGetters({
+      loggedInUser: 'users/loggedInUser',
+      currentGrant: 'grants/currentGrant',
+    }),
+    followBtnLabel() {
+      return this.userIsFollowing ? 'Following' : 'Follow';
+    },
+    followBtnVariant() {
+      return this.userIsFollowing ? 'success' : 'primary';
+    },
+    followSummaryText() {
+      const userIsFollower = this.followers.some((follower) => follower.user.id === this.loggedInUser?.id);
+      const firstFollowerName = userIsFollower ? 'you' : this.followers[0].user.name;
+
+      let otherFollowerText = '';
+      if (this.followers.length > 1) {
+        const otherFollowersCount = this.followers.length - 1;
+
+        if (otherFollowersCount === 1) {
+          otherFollowerText += ' and 1 other';
+        } else if (otherFollowersCount < 50) {
+          otherFollowerText += ` and ${otherFollowersCount} others`;
+        } else if (otherFollowersCount > 50) {
+          otherFollowerText += ' and 50+ others';
+        }
+      }
+
+      return `Followed by ${firstFollowerName}${otherFollowerText}`;
+    },
+    showFollowSummary() {
+      return this.followers.length > 0;
+    },
+    showNotesSummary() {
+      return this.notes.length > 0;
+    },
+    notesSummaryText() {
+      let textCount = `${this.notes.length}`;
+      if (this.notes.length > 50) {
+        textCount = '50+';
+      }
+
+      return `${textCount} notes`;
+    },
+  },
+  async beforeMount() {
+    this.fetchFollowState();
+    this.fetchFollowersState();
+    this.fetchNotes();
+  },
+  methods: {
+    ...mapActions({
+      getFollowerForGrant: 'grants/getFollowerForGrant',
+      getFollowersForGrant: 'grants/getFollowersForGrant',
+      getNotesForGrant: 'grants/getNotesForGrant',
+      followGrantForCurrentUser: 'grants/followGrantForCurrentUser',
+      unfollowGrantForCurrentUser: 'grants/unfollowGrantForCurrentUser',
+    }),
+    async fetchFollowState() {
+      const result = await this.getFollowerForGrant({ grantId: this.currentGrant.grant_id });
+      this.userIsFollowing = Boolean(result);
+    },
+    async fetchFollowersState() {
+      const result = await this.getFollowersForGrant({ grantId: this.currentGrant.grant_id });
+
+      if (result) {
+        this.followers = result.followers;
+      }
+    },
+    async fetchNotes() {
+      const result = await this.getNotesForGrant({ grantId: this.currentGrant.grant_id });
+
+      if (result) {
+        this.notes = result.notes;
+      }
+    },
```

Comment:
These should both be fine for the purposes of #3406, but just in case it's relevant, I want to note that `fetchNotes()` will eventually (per #3428) need to deal with pagination. 

Same thing potentially goes for `fetchFollowersState()` as well, depending on how #3407 is accomplished.

Reply:
Not sure how next steps will be implemented just yet, but added support for query params